usr/iscsiadm.c

```diff
@@ -735,7 +745,7 @@ static int print_nodes(struct iscsi_context *ctx, int info_level)
 		goto out;
 
 	if (info_level == 1)
-		print_nodes_tree(nodes, node_count);
+		print_nodes_tree(nodes, node_count, false /* not iface mode */);
```

Comment:
Comments like this, i.e. saying what variable is being set, are fine for development but do not belong in production code IMHO.

Reply:
Just save some time in the future when I look back on that line of code wondering what does `false` mean.
I you mind, I may change to enum which hold both meaning and value.